scripts/style_json.js

```diff
@@ -0,0 +1,7 @@
+import { copyFile, writeFile, mkdir } from "fs/promises";
+
+import { buildLayers } from "../src/layer/index.js";
+
+export function generateStyleJSON() {
+  writeFile("dist/style.json", JSON.stringify(buildLayers()));
```

Comment:
`buildLayers()` only generates the list of layers, not the full style. Any real use case for this JSON would require the full style. Call `buildStyle()` instead, but have that method take a base URL as a parameter instead of hardcoding `window.location`. The base URL can also come from a command line argument.